.github/actions/core-cicd/deployment/deploy-jfrog/README.md

```diff
@@ -32,7 +32,7 @@ on:
 
 jobs:
   deploy:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
```

Comment:
Suggest updating this to reflect how you've implemented with fallback, or even with no fallback at all

```suggestion
    runs-on: ubuntu-${{ vars.UBUNTU_RUNNER_VERSION || '22.04' }}
```